pootle/apps/pootle_statistics/models.py

```diff
@@ -399,8 +402,10 @@ def save(self, *args, **kwargs):
 
         super(ScoreLog, self).save(*args, **kwargs)
 
-        self.user.score += self.score_delta
-        self.user.save()
+        User = get_user_model()
+        User.objects.filter(id=self.user.id).update(
+            score=operator.add(F('score'), self.score_delta)
+        )
```

Comment:
According to https://docs.djangoproject.com/en/1.7/ref/models/queries/ the following should be valid (to be tested):

```python
        self.user.score = F('score') + self.score_delta
        self.user.save()
```